dnase/parseSamtoolsGenotypesToBedFiles.pl

```diff
@@ -166,6 +175,20 @@ sub parseGenotype {
         if (($GT1 ne $GT2) or ($GQ1 != $GQ2)) {
             die "Wrong parsing assumptions for VCF genotype ($genotype)\n";
         }
+    } elsif ($format eq "GT:GL:GQ:FT:RCL:RC:RCR:RDCN:DR:DV:RR:RV") {
+        my ($GT1,$GL1,$GQ1,$FT1,$RCL1,$RC1,$RCR1,$RDCN1,$DR1,$DV1,$RR1,$RV1) = split /\:/, $genotype;
+        ## bcftools => FORMAT=<ID=DP,Number=1,Type=Integer,Description="Number of high-quality bases">
+        #$readDepth = $DP;
+        ## delly => FORMAT=<ID=RC,Number=1,Type=Integer,Description="Raw high-quality read counts or base counts for the SV"
+        # FIXME: Make sure this is correct, there are other paramaters related to depth
+        $readDepth = $RC1;
```

Comment:
Let's use the PE tag, the same as the bcftools filter command in callSnpsMerge.sh 

$readDepth = $PE;

Reply:
I will update this.

Reply:
looks like you set $readDepth = undef;

Does using $PE not work?

Reply:
Maybe I misunderstood how the script works.
PE is an INFO tag, it looked to me like that function was parsing FORMAT tags.
I added a statement to check if the depth is undefined, if so it uses PE from the format tag,

Reply:
no you're right, that won't work. undef is fine then